src/zenml/zen_stores/rest_zen_store.py

```diff
@@ -2832,27 +2830,12 @@ def create_stack(self, stack: StackRequest) -> StackResponse:
         Returns:
             The registered stack.
         """
-        return self._create_workspace_scoped_resource(
-            resource=stack,
-            route=STACKS,
-            response_model=StackResponse,
-        )
-
-    def create_full_stack(self, full_stack: FullStackRequest) -> StackResponse:
-        """Register a full-stack.
-
-        Args:
-            full_stack: The full stack configuration.
-
-        Returns:
-            The registered stack.
-        """
-        assert full_stack.workspace is not None
+        assert stack.workspace is not None
 
         return self._create_resource(
```

Comment:
Why not `self._create_workspace_scoped_resource` as for the previous stack request?

Reply:
Exactly, due to the requirements of the 1-click deployment, I have decided to use a non-workspace scoped model.